src/main/java/com/checkout/instruments/beta/create/CreateInstrumentBankAccountRequest.java

```diff
@@ -72,7 +73,7 @@ protected CreateInstrumentBankAccountRequest(final AccountType accountType,
         this.bban = bban;
         this.swiftBic = swiftBic;
         this.currency = currency;
-        this.country = country;
+        this.country = CountryCode.fromString(country);
```

Comment:
Since this is a `beta` class, please change the constructor parameter to `CountryCode`.